sources/packages/backend/apps/api/src/route-controllers/student/student.controller.service.ts

```diff
@@ -64,24 +69,35 @@ export class StudentControllerService {
     groupName: string,
     auditUserId: number,
   ): Promise<FileCreateAPIOutDTO> {
-    const createdFile = await this.fileService.createFile(
-      {
-        fileName: file.originalname,
-        uniqueFileName: uniqueFileName,
-        groupName: groupName,
-        mimeType: file.mimetype,
-        fileContent: file.buffer,
-      },
-      studentId,
-      auditUserId,
-    );
-
+    try {
+      await this.fileService.createFile(
+        {
+          fileName: file.originalname,
+          uniqueFileName: uniqueFileName,
+          mimeType: file.mimetype,
+          fileContent: file.buffer,
+          groupName: groupName,
+        },
+        studentId,
+        auditUserId,
+      );
+    } catch (error: unknown) {
+      this.logger.error(parseJSONError(error));
+      if (error instanceof CustomNamedError) {
+        throw new InternalServerErrorException(error.message, {
+          cause: error.objectInfo,
```

Comment:
The cause will be exposed to the API client, right?
Can we prevent it since the log is logged?